### PR TITLE
Added lots of small things

### PR DESCRIPTION
**Pull request by 2m**
Avatar icons
Review and last action icons
Border around avatars that indicate review completion or request of changes

### CODE REVIEW

File: src/main/scala/akka/minion/App.scala [hunk 1]
```diff
@@ -52,17 +52,11 @@ object App {
       )
 
       system.actorOf(App.props(settings), "minion-supervisor")
-
-      println("Press Enter to stop")
```
Comment (drewhk) [inline src/main/scala/akka/minion/App.scala]: why? This was useful.

File: src/main/scala/akka/minion/App.scala [hunk 2]
```diff
@@ -52,17 +52,11 @@ object App {
...
-      StdIn.readLine()
     } catch {
       case e: Throwable =>
         println("Minion lost.")
         e.printStackTrace()
-    } finally {
-      system.terminate()
```
Comment (drewhk) [inline src/main/scala/akka/minion/App.scala]: Why?

File: src/main/scala/akka/minion/Dashboard.scala [hunk 3]
```diff
@@ -11,24 +13,51 @@ object Dashboard {
...
+
+  trait PrValidationStatus
+  object PrValidationStatus {
+    case object Success extends PrValidationStatus
+    case object Pending extends PrValidationStatus
+    case object Failure extends PrValidationStatus
+  }
+
+  case class Person(login: String, avatarUrl: String)
+  case class Performance(person: Person, action: Action, performedAt: ZonedDateTime)
```
Comment (drewhk) [inline src/main/scala/akka/minion/Dashboard.scala]: weird name?

File: src/main/scala/akka/minion/Dashboard.scala [hunk 4]
```diff
@@ -11,24 +13,51 @@ object Dashboard {
...
+
+  case class ApiUsageStats(limit: Int, remaining: Int, resetsIn: String)
```
Comment (drewhk) [inline src/main/scala/akka/minion/Dashboard.scala]: this is nice to have yes

File: src/main/scala/akka/minion/GithubService.scala [hunk 5]
```diff
@@ -1,35 +1,58 @@
...
+import akka.stream.{ActorMaterializer, OverflowStrategy, ThrottleMode}
+import spray.json._
 
-import scala.concurrent.{Future, Promise}
-import scala.concurrent.duration._
 import scala.collection.immutable.Seq
+import scala.concurrent.duration._
+import scala.concurrent.{Future, Promise}
+
+trait ZuluDateTimeMarshalling {
```
Comment (drewhk) [inline src/main/scala/akka/minion/GithubService.scala]: Great, thanks for implementing this!

File: src/main/scala/akka/minion/HttpServer.scala [hunk 6]
```diff
@@ -225,4 +242,102 @@ object Template {
     )
   }
 
+  def involvment(p: Performance): TypedTag[String] = {
```
Comment (drewhk) [inline src/main/scala/akka/minion/HttpServer.scala]: involvement

File: src/main/scala/akka/minion/Dashboard.scala (hunk 3 above)
Comment (2m) [inline src/main/scala/akka/minion/Dashboard.scala]: Yea. Late evening bad naming decision. I first thought of it as a performance of an actor. But now I clearly see the weirdness that this word brings.

**Pull request merged**